ts/src/base/Exchange.ts

```diff
@@ -1601,6 +1611,36 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchTradesWs() is not supported yet');
     }
 
+    async watchLiquidations (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}) {
+        throw new NotSupported (this.id + ' watchLiquidations() is not supported yet');
+    }
+
+    async watchAllLiquidations (symbols: string[], since: Int = undefined, limit: Int = undefined, params = {}) {
```

Comment:
@pcriadoperez should we rename it to `watchLiquidationsForSymbols` to make it compatible with the other implementations?

Reply:
nevermind just saw you already have it